fv/test-workload/test-workload.go

```diff
@@ -171,10 +171,19 @@ func main() {
 					return
 				}
 				err = utils.RunCommand("ip", "-6", "route", "add", "default", "via", hostIPv6Addr.String(), "dev", "eth0")
+				if err != nil {
```

Comment:
Should probably log and carry on in this file, it's just collecting diags and there's no reason to skip later diags because one  failed.

Reply:
Fixed